spec/directives/import/configuration.hrx

```diff
@@ -0,0 +1,135 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#3040
+
+<===>
+================================================================================
+<===> same-file/input.scss
+$d-a: configured;
+@import "other";
+
+<===> same-file/_other.scss
+$a: original !default;
+b {c: $a}
+
+<===> same-file/_other.import.scss
+@forward "other" as d-*;
+
+<===> same-file/output.css
+b {
+  c: configured;
+}
+
+<===>
+================================================================================
+<===> separate-file/input.scss
+@import "config";
+@import "configurable";
+
+<===> separate-file/_config.scss
+$d-a: configured;
+
+<===> separate-file/_configurable.scss
+$a: original !default;
+b {c: $a}
+
+<===> separate-file/_configurable.import.scss
+@forward "configurable" as d-*;
+
+<===> separate-file/output.css
+b {
+  c: configured;
+}
+
+<===>
+================================================================================
+<===> import-twice/no-change/input.scss
+$d-a: configured;
+@import "other";
+@import "other";
+
+<===> import-twice/no-change/_other.scss
+$a: original !default;
+b {c: $a}
+
+<===> import-twice/no-change/_other.import.scss
+@forward "other" as d-*;
+
+<===> import-twice/no-change/output.css
+b {
+  c: configured;
+}
+
+b {
+  c: configured;
+}
+
+<===>
+================================================================================
+<===> import-twice/with-change/input.scss
```

Comment:
I think you also want a test that verifies that, after the second import, the variable does have the new value even though the module hasn't been reloaded.